src/wp-includes/blocks.php

```diff
@@ -723,31 +723,31 @@ function get_dynamic_block_names() {
 }
 
 /**
- * Retrieves block types hooked into the given block, grouped by their relative position.
+ * Retrieves block types hooked into the given block, grouped by anchor block type and the relative position.
  *
  * @since 6.4.0
  *
- * @param string $name Block type name including namespace.
- * @return array[] Array of block types grouped by their relative position.
+ * @return array[] Array of block types grouped by anchor block type and the relative position.
  */
-function get_hooked_blocks( $name ) {
-	$block_types   = WP_Block_Type_Registry::get_instance()->get_all_registered();
-	$hooked_blocks = array();
+function get_hooked_blocks() {
+	$block_types = WP_Block_Type_Registry::get_instance()->get_all_registered();
+	$block_hooks = array();
```

Comment:
Nitpick, should we not keep the variable name? Since we're returning this variable from a function called `get_hooked_blocks`, I thought it made sense to call it `$hooked_blocks` 😅 

(BTW the naming was deliberate -- it seemed like what this function does is indeed return the blocks that are hooked to a given anchor block, whereas the block hook is specified by that anchor block plus relative position.)

Reply:
Addressed with https://github.com/WordPress/wordpress-develop/pull/5399/commits/21efb9edb6f4ef1881cdb6bed8fa1c97c65ba0f6.